### DISCUSSION TITLE
Is there a way to edit a loved track?

### DISCUSSION THREAD

**Discussion by ghost** [RudeySH/lastfm-bulk-edit#13] [Ideas]
I have a track I loved ages ago, and realized it had the wrong title. So I edited all scrobbles of that song, but now the loved track is still pointing to the same track, but now it says I have 0 scrobbles of that track.

Not sure if it's within the scope of this project. Right now I'm just hoping to get some tips to solve this issue.

**Comment by RudeySH** [comment]
After editing the track name or artist name of a track, Last.fm thinks the result is a different track from what is was before. And so the new track is not loved yet.

To fix it, you'll have to click the heart to love the new track. (And unlove the old track.)

**Comment by ghost** [comment]
Sorry for not clarifying, the main issue is that I don't want to lose the date that I loved the track. I want to keep the fact that I loved it like 7 years ago.

**Reply by RudeySH** [reply]
Unfortunately, that's not possible. It would be best to keep the track name as-is, then.
